src/main/webapp/i18n/de/global.json

```diff
@@ -188,6 +188,7 @@
             "editInEditor": "In Editor bearbeiten",
             "editInIntelliJ": "In IntelliJ bearbeiten",
             "editEntity": "{{entity}} bearbeiten",
+            "exampleSubmissions": "Beispielabgaben",
```

Comment:
Is it Beispielabgaben or Beispiellösungen as in `src/main/webapp/i18n/de/exampleSubmission.json`?

Reply:
Thank you @lschlesinger for having a sharp eye on these terms.
I used Beispielabgaben for the example submissions.
Beispiellösung would be an **example solution**, which is another concept in Artemis, out of this PR scope.